restful_api/tests/test_access_token_expires_in.py

```diff
@@ -0,0 +1,10 @@
+from odoo.tests.common import SingleTransactionCase
+
+
+class TestTokenExpiry(SingleTransactionCase):
+    """Test token expiration."""
+
+    def test_access_token_expires_in(self, arg):
+        """Test default token expiration date in seconds."""
+        expires_in = self.env.ref("restful_api.access_token_expires_in")
+        self.assertEqual(expires_in.value, 31536000)
```

Comment:
Better to name this as a variable e.g `one_year`, instead of leaving it as a magic number.

Reply:
This part implement to auto api testing purpose, it’s in pending,test folder has been removed now.